app/data_model/list_store.py

```diff
@@ -0,0 +1,82 @@
+import random
+from string import ascii_letters
+from collections import OrderedDict
+
+from structlog import get_logger
+
+from app.settings import EQ_LIST_ITEM_ID_LENGTH
+
+logger = get_logger()
+
+
+def random_string(length):
+    return ''.join(random.choice(ascii_letters) for _ in range(length))
+
+
+class ListStore:
+    """ Storage for lists.
+    """
+
+    def __init__(self, existing_items=None):
+        if not existing_items:
+            existing_items = []
+
+        self._lists = OrderedDict(existing_items)
```

Comment:
We aren't using any of the features that OrderedDict offers over a standard dict in 3.7, e.g. ordered equality comparison or `move_to_end`. This should be a standard Dict.

Reply:
We do need to preserve the order of these elements, and I was under the impression that python dictionaries shouldn't be used as ordered dicts (even though they work) since the ordering is not a language feature, just an implementation detail?

Reply:
I'm out of date. Looks like this is now a language feature in 3.7. I'll switch it over.